be/src/vec/exec/jni_connector.cpp

```diff
@@ -62,6 +62,21 @@ JniConnector::~JniConnector() {
     }
 }
 
+Status JniConnector::open() {
```

Comment:
call `open(nullptr, nullptr)` directly

Reply:
Thanks for your suggestion, I have modified here